application.go

```diff
@@ -119,6 +119,7 @@ func (a *Application) Run() error {
 		glfw.WindowHint(glfw.Visible, glfw.False)
 	}
 
+	glfw.WindowHint(glfw.ScaleToMonitor, glfw.True)
 	if a.config.windowAlwaysOnTop {
```

Comment:
It comes from this line

Reply:
I just fix it by change my window size  in options